app/components/task-details.hbs

```diff
@@ -1,12 +1,46 @@
-<div class="task-details">
-  <p class="task-details__title"><b>Title: </b>{{@task.title}}</p>
-  <p class="task-details__purpose"><b>Purpose: </b>{{@task.purpose}}</p>
-  <div class="dates-info">
-    <div class="dates-info__end">
-      <p><b>Deadline: </b>{{convertDate @task.endsOn end_date=1}}</p>
+<div class='task-details'>
+  <div class='task-details__title-container'>
+    <p class='task-details__title'>{{@task.title}}</p>
+    <div class='task-card__progress-bar-container'>
+      <div class='task-card__progress-bar-container__progress-bar'>
+        <input
```

Comment:
add `label` element in the input for good practices

Reply:
input is used as range here. I don't see a point of adding a label tag without any text.